packages/host/app/components/operator-mode/interact-submode.gts

```diff
@@ -278,6 +279,13 @@ export default class InteractSubmode extends Component<Signature> {
         }
         await changeSizeCallback();
       },
+      doCloseAnimation: async (card: CardDef) => {
```

Comment:
why not just pass the stackItem in instead of the cardDef? then you won't need the `findCardInStack()`

Reply:
Changing back to the `CardDef` type of argument as `StackItem` type from host should not be imported into runtime-common ( mentioned [here](https://discord.com/channels/584043165066199050/900021750367129641/1296649575288930364) )